lib/briefly/entry.ex

```diff
@@ -124,4 +123,13 @@ defmodule Briefly.Entry do
   defp extname(%{extname: value}), do: value
   defp extname(_), do: Briefly.Config.default_extname
 
+  defp cleanup(ets, pid) do
+    case :ets.lookup(ets, pid) do
+      [{^pid, _tmp, paths}] ->
```

Comment:
@bruce @benwilson512 I believe pattern matching on `pid` here is the original intent but the previous code missed the caret.